tests/Neo.Plugins.DBFTPlugin.Tests/ConsensusTestHelper.cs

```diff
@@ -0,0 +1,303 @@
+            {
+                service.Tell(payload);
+            }
+        }
+
+        /// <summary>
+        /// Sends a message to specific consensus services
+        /// </summary>
+        public void SendToValidators(ExtensiblePayload payload, IActorRef[] consensusServices, int[] validatorIndices)
+        {
+            foreach (var index in validatorIndices)
+            {
+                if (index >= 0 && index < consensusServices.Length)
+                {
+                    consensusServices[index].Tell(payload);
+                }
+            }
+        }
+
+        /// <summary>
+        /// Simulates a complete consensus round
+        /// </summary>
+        public void SimulateCompleteConsensusRound(IActorRef[] consensusServices, uint blockIndex = 1, UInt256[] transactions = null)
+        {
+            var validatorCount = consensusServices.Length;
+            var primaryIndex = (int)(blockIndex % (uint)validatorCount);
+
+            // Step 1: Primary sends PrepareRequest
+            var prepareRequest = CreatePrepareRequest(transactionHashes: transactions);
+            var prepareRequestPayload = CreateConsensusPayload(prepareRequest, primaryIndex, blockIndex);
+            SendToAll(prepareRequestPayload, consensusServices);
+
+            // Step 2: Backup validators send PrepareResponse
+            for (int i = 0; i < validatorCount; i++)
+            {
+                if (i != primaryIndex) // Skip primary
+                {
+                    var prepareResponse = CreatePrepareResponse();
+                    var responsePayload = CreateConsensusPayload(prepareResponse, i, blockIndex);
+                    SendToAll(responsePayload, consensusServices);
+                }
+            }
+
+            // Step 3: All validators send Commit
+            for (int i = 0; i < validatorCount; i++)
+            {
+                var commit = CreateCommit();
+                var commitPayload = CreateConsensusPayload(commit, i, blockIndex);
+                SendToAll(commitPayload, consensusServices);
+            }
+        }
+
+        /// <summary>
+        /// Simulates a view change scenario
+        /// </summary>
+        public void SimulateViewChange(IActorRef[] consensusServices, int[] initiatingValidators, byte newViewNumber, ChangeViewReason reason = ChangeViewReason.Timeout)
+        {
+            foreach (var validatorIndex in initiatingValidators)
+            {
+                var changeView = CreateChangeView(reason);
+                var changeViewPayload = CreateConsensusPayload(changeView, validatorIndex, viewNumber: newViewNumber);
+                SendToAll(changeViewPayload, consensusServices);
+            }
+        }
+
+        /// <summary>
+        /// Simulates Byzantine behavior by sending conflicting messages
+        /// </summary>
+        public void SimulateByzantineBehavior(IActorRef[] consensusServices, int byzantineValidatorIndex, uint blockIndex = 1)
```

Comment:
This is very interesting, we did not have this scenario in the past being tested. But here we still have lots of work to do to send this payloads and monitor the behavior of the test. But it is good to see that you pushed this scenario.

Reply:
Please tell me what you want or need, i can add them and update this pr for you.